### PR TITLE
Add Docs

### PR DESCRIPTION
**Pull request by geonnave**
For now just updating the README, adding a `.gitignore`, and adding a print statement to the example `coapserver.rs`.

@malishav, it looks like libraries in Rust should not have `Cargo.lock` checked into version control, can you confirm that? (if it is the case, I will add it to the `.gitignore` as well).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,29 @@ Supports:
...
+cargo build
+cargo test
+```
 
 ## Example
 
-TODO
+In one terminal, run an example CoAP server:
+```
+./target/debug/coapserver
```
Comment (malishav) [inline README.md:25]: Let's use `cargo run --bin` for this

File: README.md [hunk 2]
```diff
@@ -9,8 +9,29 @@ Supports:
 
 ## Installation
 
-TODO
+1. Make sure you have Rust installed (see instructions in the [website](https://www.rust-lang.org/tools/install)).
+
+2. Download, compile, and run the tests:
+```
+git clone git@github.com:geonnave/edhoc-rs.git && cd edhoc-rs
```
Comment (malishav) [inline README.md:16]: Let's use the official repo here, `openwsn-berkeley/edhoc-rs`

File: README.md [hunk 3]
```diff
@@ -9,8 +9,29 @@ Supports:
...
 
-TODO
+In one terminal, run an example CoAP server:
+```
+./target/debug/coapserver
+```
+
+In another one, run the client:
+```
+./target/debug/coapclient
```
Comment (malishav) [inline README.md:30]: Same comment as above

**Review by malishav — COMMENTED** [review]
Thanks @geonnave for providing the PR !

**Comment by malishav** [comment]
> @malishav, it looks like libraries in Rust should not have Cargo.lock checked into version control, can you confirm that? (if it is the case, I will add it to the .gitignore as well).

Yes, I think we can add `Cargo.lock` to `.gitignore`

**Comment by geonnave** [comment]
I think the README docs are enough for now, since many things are likely to change, such as the build system parameters.

**Comment by malishav** [comment]
Looks great! Let's just add steps on building and flashing for nrf52840 with and without hardware support for now. That should be pretty useful for a newcomer to the repo.

**Comment by geonnave** [comment]
Good idea, done.

**Comment by malishav** [comment]
Looks great, merging

**Pull request merged**